templates/report_template.html

```diff
@@ -72,13 +73,22 @@
 							<td style="text-align: center;" bgcolor="#3465a4"><a href="https://www.youtube.com/watch?v=68ugkg9RePc" style="text-decoration: none; color: #000;">{{row.lcb_result}}</a></td>
 							<td style="text-align: center;"><p>N/A</p></td>
 							<td style="text-align: center;"><p>N/A</p></td>
+							<td style="text-align: center;"><p>N/A</p></td>
 
 						<!-- LCB is "UNSTABLE" -->
 						{% elif row.lcb_result == "UNSTABLE" %}
 							<td style="text-align: center;"><a href="{{row.lcb_url}}">{{row.lcb_num}}</a></td>
 							<td style="text-align: center;">{{row.build_days_ago}}</td>
 							<td style="text-align: center;">{{ composes(row) }}</td>
 							<td style="text-align: center;" bgcolor="#ffb738">{{row.lcb_result}}</td>
+							<td style="text-align: center;"><p>{{ row.stage_name }}</p>
+							{% if row.stage_urls is not none %}
```

Comment:
Shouldn't it also be N/A in case of unstable?

Reply:
Yes, you are right. I have checked it and if the state is unstable then it is problem with tests. They are already linked in the table, so I am going to set this to N/A as you suggested.